src/api/api.ts

```diff
@@ -16,6 +16,7 @@ import {
 import { formatAuthParams } from './api.utils';
 import { Config } from './config';
 import { debugLog } from './debug-log';
+import getenv from 'getenv';
```

Comment:
Not used (probably used in `debugLog` function)

Reply:
It's used in the timeout function (see above comment)